src/architect/project/project.utils.ts

```diff
@@ -0,0 +1,229 @@
+Go into '${name}', then run the register and deploy commands
+  % architect register .
+  % architect deploy ${name}`;
+  }
+
+  private static generateMultiCoponentReadme(root_service: string, secondary_service: string) {
+    return `# This project was auto-generated using the Architect CLI.
+### To run your project locally:
+Go into '${secondary_service}', then run the link and link commands.
+  % architect link .
+Change directory to '../${root_service}' and run the dev command.
+  % architect dev .
+### To deploy your project to the Cloud:
+Go into '${secondary_service}' and run the register command.
+  % architect register .
+Change directory to '../${root_service}', then run the register and deploy commands
+  % architect register .
+  % architect deploy ${root_service}`;
+  }
+
+  static async updateArchitectYamls(app: AppService, selections: Dictionary<any>, project_dir: string): Promise<void> {
+    const backend = selections['backend'];
+    const backend_yml_path = `./${project_dir}/${backend.name.toLowerCase()}/architect.yml`;
+    const backend_yml = yaml.load(fs.readFileSync(backend_yml_path).toString('utf-8')) as ComponentSpec;
+    const database_yml = await this.fetchYamlFromGitHub(selections['database']['architect-file']);
+    this.switchDatabase(database_yml, backend_yml, backend_yml_path);
+
+    // Need better handling for when a frontend requires a backend.
+    const frontend = selections['frontend'];
+    if (frontend) {
+      const frontend_yml_path = `./${project_dir}/${frontend.name.toLowerCase()}/architect.yml`;
+      const frontend_yml = yaml.load(fs.readFileSync(frontend_yml_path).toString('utf-8')) as ComponentSpec;
+      const backend_component_name = backend ? backend.repository.split('/').pop() : undefined;
+      await this.updateDependencies(frontend_yml, frontend_yml_path, backend_component_name);
+    }
+
+    let readme: string;
+    if (!frontend && backend) {
+      readme = this.generateSingleComponentReadme(backend.name.toLowerCase());
+    } else {
+      readme = this.generateMultiCoponentReadme(frontend.name.toLowerCase(), backend.name.toLowerCase());
+    }
+    fs.writeFileSync(`./${project_dir}/README.md`, readme);
+  }
+
+  static async getSelections(): Promise<Dictionary<Selection>> {
+    // get choices from template-configs repository
+    const config_file = 'https://raw.githubusercontent.com/architect-team/template-configs/main/config.json';
+    const config_json = await this.fetchJsonFromGitHub(config_file) as Dictionary<any>;
+    const choices = config_json.choices;
+
+    const selections: Dictionary<Selection> = {};
+    const types = ['Full stack', 'Backend'];
+    const type = await this.prompt(types, 'Select a type you would like to proceed');
+    if (type.toLowerCase() === 'full stack') {
+      const frontend_opts = choices.filter((item: any) => item.type === 'frontend');
+      const frontend = await this.prompt(frontend_opts, 'Select a frontend');
```

Comment:
We can disable it if you'd like.

Reply:
My thoughts: 

- for V1 we'll disable it. You'll select your stack, and since we only have one selection for Front/backend, it will just default to those
- For V2 after we get more options to choose from, we'll add this feature back in to allow the user to select what they want